dev/com.ibm.ws.microprofile.faulttolerance.2.0/test/src/com/ibm/ws/microprofile/faulttolerance/test/AbstractFTTest.java

```diff
@@ -25,7 +29,9 @@
     @Before
     public void before() {
         System.setProperty(FTConstants.JSE_FLAG, "true");
-        FaultToleranceProviderResolver.setInstance(new ProviderResolverImpl20());
+        ProviderResolverImpl20 providerResolver = new ProviderResolverImpl20();
```

Comment:
Should this be @Before or @BeforeClass? Does it matter?

Reply:
I think it could be either, as long is it matches up with the `after` method.

I don't think it's important as it's not doing anything expensive.